login-screen.html

```diff
@@ -0,0 +1,11 @@
+<html>
+
+<head>
+  <title>Login screen</title>
```

Comment:
Change this to 'The Amazing Login Screen'

Reply:
Looks cool

Reply:
Can you also add a ! or 2?

Reply:
@Gabriel-Dalmoro Really great point. Didn't think of that. That will make this much better.

Reply:
Yes, never see this kind before!